src/Microsoft.ML.Data/EntryPoints/EntryPointNode.cs

```diff
@@ -935,7 +929,7 @@ private object BuildParameterValue(List<ParameterBinding> bindings)
                         ch.Warning("Node '{0}' has unexpected fields that are ignored: {1}", id, string.Join(", ", unexpectedFields.Select(x => x.Name)));
                     }
 
-                    result.Add(new EntryPointNode(env, ch, catalog, context, id, nodeName, inputs, outputs, checkpoint, stageId, cost, label, group, weight, name));
+                    result.Add(new EntryPointNode(env, ch, context, id, nodeName, inputs, outputs, checkpoint, stageId: stageId, cost: cost, label: label, group: group, weight: weight, name: name));
```

Comment:
Are the named parameters at the end of this line necessary?  `stageId: stageId, cost: cost, label: label, group: group, weight: weight, name: name));`